tests/ballerina-test-utils/src/main/java/org/ballerinalang/test/util/BCompileUtil.java

```diff
@@ -409,8 +414,20 @@ private static CompileResult compileOnJBallerina(String sourceFilePath) {
 
         BLangPackage bLangPackage = (BLangPackage) compileResult.getAST();
         byte[] compiledJar = JVMCodeGen.generateJarBinary(bLangPackage, context, packageName);
-        compileResult.setCompiledJarFile(compiledJar);
         compileResult.setEntryClassName(FileUtils.cleanupFileExtension(packageName));
```

Comment:
setEntryClassName and getEntryClassName methods are not needed anymore right? they are used within this method itself